hydromt_fiat/workflows/gis.py

```diff
@@ -31,14 +36,28 @@ def get_area(gdf: gpd.GeoDataFrame) -> gpd.GeoDataFrame:
     if gdf.crs.is_geographic:
         # If the CRS is geographic, reproject to the nearest UTM zone
         nearest_utm = utm_crs(gdf.total_bounds)
+        unit = nearest_utm.axis_info[0].unit_name
         gdf_utm = gdf.to_crs(nearest_utm)
         gdf["area"] = gdf_utm["geometry"].area
     elif gdf.crs.is_projected:
         # If the CRS is projected, calculate the area in the same CRS
+        unit = gdf.crs.axis_info[0].unit_name
         gdf["area"] = gdf["geometry"].area
+    
+    ureg = pint.UnitRegistry()
+    unit = ureg(unit).units
+    model_unit = ureg(model_length_unit).units
+    
+    if unit != model_unit:
+        if model_unit == Units.meters.value and unit == Units.feet.value:
+            gdf["area"] = gdf["area"] * math.exp(Conversion.feet_to_meters)
```

Comment:
why use the exponential for conversion? This will be e^0.3048. You want 0.3048^2, so this should be 
```suggestion
            gdf["area"] = gdf["area"] * Conversion.feet_to_meters**2
```

Same below for the other conversion